### PR TITLE
Add Beauty Queen

### PR DESCRIPTION
**Pull request by GingertronMk1**

### CODE REVIEW

File: _shows/18_19/beauty_queen_of_leenane.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  name: Laura Finkch
+- role: Production Assistant
+  name: Caitlin Masterson
+- role: Set Designer
+  name: Tom Proffitt
+- role: Set Assistant
+  name: Francesca Ashby
+- role: Set Construction
+  name: Jack Ellis
+- role: Lighting Designer
```
Comment (samozzy) [inline _shows/18_19/beauty_queen_of_leenane.md]: Technical Director: me 😜

Comment (nhyty) [reply _shows/18_19/beauty_queen_of_leenane.md]: You sure :stuck_out_tongue:

Comment (GingertronMk1) [reply _shows/18_19/beauty_queen_of_leenane.md]: He ain't in the program

File: _shows/18_19/beauty_queen_of_leenane.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- role: Poster Designer
+  name: Jase Neal
+- role: Technical Operator
+  name: Zoe Smith
+- role: Technical Operator
+  name: Izzy de Bono
+- role: Technical Operator
+  name: Martin Tomlinson
+- role: Technical Operator
+  name: Luwa Adebanjo
```
Comment (samozzy) [inline _shows/18_19/beauty_queen_of_leenane.md]: Replace with Holly Jessop

**Pull request merged**